Coupled-Cluster/Spin_Orbitals/CCSD/helper_CC.py

```diff
@@ -2,7 +2,7 @@
 # Scipy and numpy python modules are required
 #
 # Algorithms were taken directly from Daniel Crawford's programming website:
-# http://github.com/CrawfordGroup/ProgrammingProjects
+# http://github.com/CrawfordGroup/ProgrammingProjects  
```

Comment:
Can you pop the space after all of these lines?

Reply:
Done!